presto-spi/src/main/java/com/facebook/presto/spi/security/RoleGrant.java

```diff
@@ -0,0 +1,86 @@
+/*
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package com.facebook.presto.spi.security;
+
+import com.fasterxml.jackson.annotation.JsonCreator;
+import com.fasterxml.jackson.annotation.JsonProperty;
+
+import java.util.Objects;
+
+import static java.util.Locale.ENGLISH;
+import static java.util.Objects.requireNonNull;
+
+public class RoleGrant
+{
+    private final PrestoPrincipal grantee;
+    private final String roleName;
+    private final boolean grantable;
+
+    @JsonCreator
+    public RoleGrant(@JsonProperty("grantee") PrestoPrincipal grantee, @JsonProperty("roleName") String roleName, @JsonProperty("grantable") boolean grantable)
```

Comment:
Use enum instead of boolean for `grantable`.
It's non clear what's the intention when you use `new RoleGrant("...", "...", false)`
`new RoleGrant("...", "...", GRANTABLE)` or `new RoleGrant("...", "...", NON_GRANTABLE)` would be much more readable.